packages/css/src/alert.css

```diff
@@ -6,25 +6,26 @@
   --dsc-alert-icon-url: url("data:image/svg+xml,%3Csvg viewBox='0 0 24 24' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill-rule='evenodd' d='M3.25 4A.75.75 0 0 1 4 3.25h16a.75.75 0 0 1 .75.75v16a.75.75 0 0 1-.75.75H4a.75.75 0 0 1-.75-.75zM11 7.75a1 1 0 1 1 2 0 1 1 0 0 1-2 0m-1.25 3a.75.75 0 0 1 .75-.75H12a.75.75 0 0 1 .75.75v4.75h.75a.75.75 0 0 1 0 1.5h-3a.75.75 0 0 1 0-1.5h.75v-4h-.75a.75.75 0 0 1-.75-.75'/%3E%3C/svg%3E");
   --dsc-alert-border-radius: var(--ds-border-radius-md);
   --dsc-alert-color: var(--ds-color-info-text-default);
-  --dsc-alert-gap: var(--ds-size-2);
+  --dsc-alert-spacing: var(--ds-size-2);
   --dsc-alert-icon-size: var(--ds-size-7);
-  --dsc-alert-padding: var(--ds-size-6);
+  --dsc-alert-padding-block: var(--ds-size-6);
 
   background: var(--dsc-alert-background);
   border-radius: var(--dsc-alert-border-radius);
-  border: 1px solid var(--dsc-alert-border-color);
+  border: 1px solid;
+  border-color: var(--dsc-alert-border-color);
   box-sizing: border-box;
   color: var(--dsc-alert-color);
-  padding-block: var(--dsc-alert-padding);
-  padding-inline: calc(var(--dsc-alert-padding) + var(--dsc-alert-icon-size) + var(--dsc-alert-gap)) var(--dsc-alert-padding);
+  padding-block: var(--dsc-alert-padding-block);
+  padding-inline: calc(var(--dsc-alert-padding-block) + var(--dsc-alert-icon-size) + var(--dsc-alert-spacing)) var(--dsc-alert-padding-block);
```

Comment:
Her er `padding-block` brukt til `padding-inline`, og dersom man sender inn `padding-block: 1rem 2rem` (som er mulig i vanlig `padding-block`) så brekker det. Skulle vi ha funnet på enda et ord her? 
eller evt. gjort om så det er:
```
--dsc-alert-padding => --dsc-alert-spacing
--dsc-alert-gap => --dsc-alert-icon-spacing
```

Reply:
For å sitere meg sjølv:
`border: now uses two properties. The color is a variable, and the width and style is a css prop.`

Denne gjelder der samme border ikkje er brukt fleire plassa. `table` fks, burde ha eigen variabel for `border:`

Reply:
Ser ut som eg har greid å lagt denne kommentaren på feil plass, eg skal sjekke dette ut.

Reply:
Eg har endra denne til å bruke eksplisitte css props, Det vil no sjå slik ut:
```css
  --dsc-alert-padding-block: var(--ds-size-6);
  --dsc-alert-padding-inline-end: var(--ds-size-6);
  
  padding-block: var(--dsc-alert-padding-block);
  padding-inline-start: calc(var(--dsc-alert-padding-inline-end) + var(--dsc-alert-icon-size) + var(--dsc-alert-spacing));
  padding-inline-end: var(--dsc-alert-padding-inline-end);
```

Eg er ikkje veldig fornøgd med løysingen, men det er bedre.